dev-libs/zziplib/zziplib-0.13.71.recipe

```diff
@@ -5,34 +5,39 @@ HOMEPAGE="http://zziplib.sourceforge.net/"
 COPYRIGHT="1999-2010 Guido Draheim"
 LICENSE="GNU LGPL v2.1
 	MPL v1.1"
-REVISION="4"
+REVISION="1"
 SOURCE_URI="https://github.com/gdraheim/zziplib/archive/v$portVersion.tar.gz"
-CHECKSUM_SHA256="846246d7cdeee405d8d21e2922c6e97f55f24ecbe3b6dcf5778073a88f120544"
+CHECKSUM_SHA256="2ee1e0fbbb78ec7cc46bde5b62857bc51f8d665dd265577cf93584344b8b9de2"
 PATCHES="zziplib-$portVersion.patchset"
 
 ARCHITECTURES="x86_gcc2 x86 x86_64"
 SECONDARY_ARCHITECTURES="x86"
 
-libVersion="13.0.69"
+libVersion="13.0.71"
 libVersionCompat="$libVersion compat >= 13"
```

Comment:
```suggestion
libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
```

Reply:
local copy already has 13.0.71, not sure why I didn't push that change :)